crates/time_format/src/time_format.rs

```diff
@@ -16,14 +16,23 @@ pub fn format_localized_timestamp(
         let timestamp_local_date = timestamp_local.date();
 
         let native_fmt = if timestamp_local_date == reference_local_date {
-            macos::format_time(&timestamp)
+            macos::format_time(&timestamp).map(|t| format!("Today at {}", t).to_string())
         } else if reference_local_date.previous_day() == Some(timestamp_local_date) {
-            macos::format_time(&timestamp).map(|t| format!("yesterday at {}", t).to_string())
+            macos::format_time(&timestamp).map(|t| format!("Yesterday at {}", t).to_string())
         } else {
-            macos::format_date(&timestamp)
+            match macos::format_time(&timestamp) {
+                Ok(time_format) => macos::format_date(&timestamp)
+                    .map(|t| format!("{} {}", t.replace('-', "/"), time_format)),
```

Comment:
Why do we do this? I also just wanted to put up PR cleaning up the notification panel formatting but our changes will have conflicts. Wanna pair on this together?

Reply:
> taken into account that the splitting delimiter character can be either / or - when formatted natively by macOS, I've defaulted it to / instead for consistency, which also compliments our current fallback mechanism. From what I've tested, Discord seems to also do this.
Would this ^ not be a plausable excuse? 

Sure we can if you're fine with me being muted and type instead. Wanna call on Zed and take it from there?